src/Multiselect/MultiselectComponents.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import classNames from 'classnames';
+import { components } from 'react-select';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+import { Close, KeyboardArrowDown } from '../../icons';
+
+const ClearIndicator = (props) => (
+  <components.ClearIndicator {...props}>
+    <Icon src={Close} />
+  </components.ClearIndicator>
+);
+
+const DropdownIndicator = (props) => (
+  <components.DropdownIndicator {...props}>
+    <Icon src={KeyboardArrowDown} />
+  </components.DropdownIndicator>
+);
+
+const ValueContainer = ({ children, ...props }) => (
+  <components.ValueContainer {...props}>
+    {React.Children.map(children, (child) => (child && child.type !== components.Placeholder ? child : null))}
+  </components.ValueContainer>
+);
+
+const Control = ({
+  isFocused,
+  hasValue,
+  children,
+  selectProps,
+  ...props
+}) => (
+  <>
+    <components.Control
+      className={classNames('pgn__multiselect__control', { 'is-focus': isFocused })}
+      {...props}
+      hasValue={hasValue}
+      isFocused={isFocused}
+    >
+      <components.Placeholder
+        className={classNames('pgn__multiselect__floating-label',
+          { show: hasValue || selectProps.inputValue || isFocused })}
+        {...props}
+      >
+        {selectProps.placeholder}
+      </components.Placeholder>
+      {React.Children.map(children, (child) => React.cloneElement(child, { ...child.props, isFocused }))}
+    </components.Control>
+  </>
+);
+
+const Menu = (props) => (
+  <components.Menu className="pgn__multiselect__menu" {...props} />
+);
+
+const Option = (props) => (
+  <components.Option className={classNames({ 'is-focus': props.isFocused })} {...props} />
+);
+
+const MultiValueContainer = ({ innerProps, ...props }) => {
```

Comment:
[curious] Did we try to explore whether we could make use of the existing `Chip` here? The `Chip`-like elements in this multiselect implementation are different from the behavior of the Paragon `Chip` component in that the entire element is clickable to dismiss it, not just the icon.

Related, is there any way to focus with keyboard only navigation on these chips for a11y support?

Reply:
Yes, `Chip` in the `Multiselect` is different from the Paragon's `Chip` and actually it was easier to change some styles to fit design vs replacing whole component. In last commit I added a11y support (focus for `Chip` icon and `onKeyPress` activates `onClick`)